appcues/src/main/java/com/appcues/debugger/DebuggerStatusManager.kt

```diff
@@ -353,7 +353,7 @@ internal class DebuggerStatusManager(
     }
 
     suspend fun checkPushValidation() {
-        if (pushStatus == SUCCESS) return
+        if (pushStatus == SUCCESS || pushStatus == LOADING) return
```

Comment:
prob ok to allow another test on SUCCESS case? if they wanted to retry and check anything about their app UI. Not a big deal but any reason to prevent that?